docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "3"
+
+services:
+  mysql:
+    image: mysql
+    environment:
+      MYSQL_USER: ${DB_USER}
+      MYSQL_PASSWORD: ${DB_PASS}
+      MYSQL_DATABASE: ${DB_NAME}
+      NYSQL_HOST: ${DB_HOST}
+      MYSQL_PORT: ${DB_PORT}
+      MYSQL_RANDOM_ROOT_PASSWORD: "true"
+    volumes:
+      - ./migrations/base.sql:/docker-entrypoint-initdb.d/init.sql:ro
+    ports:
```

Comment:
defining ports like this is useless, by default all containers are in the same network in the same compose

Reply:
it could be useful to access to the db from outside the container